packages/flutter_tools/templates/plugin/pubspec.yaml.tmpl

```diff
@@ -27,11 +27,25 @@ flutter:
   # adding or updating assets for this project.
   plugin:
     platforms:
+{{#no_platforms}}
+    # You see this message because this plugin project was generated without specifying any
```

Comment:
"You see this message because" is kind of clunky.
What about "This plugin project..."